contracts/contracts/levels/SafeToken.sol

```diff
@@ -0,0 +1,15 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import '@openzeppelin/contracts/token/ERC20/ERC20.sol';
```

Comment:
Shouldn't this import be similar to the one of `SafeTokenBackdoor` ?

Reply:
Yes both are basically, same, we have 
`import 'openzeppelin-contracts-08/token/ERC20/ERC20.sol';` and
`import '@openzeppelin/contracts/token/ERC20/ERC20.sol';`

To keep things consistent, we will keep it the same.